tsconfig.json

```diff
@@ -1,11 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "target": "ES5",
```

Comment:
is this necessary? can you revert this change and see if things are fine? sometimes you just need to restart TS Server on VSCode is lint errors come up.

Reply:
Same with origin. only changed the text transform, but I'll change to lowercase.

Reply:
Thanks